tests/unit/components/playback/controls/SettingsFormVuex.spec.js

```diff
@@ -0,0 +1,203 @@
+      user_id: 0,
+      uuid: "2VSckkkkk2An3dqHEyfRRE",
+      nickname: "Lab User -1",
+    },
+    {
+      user_id: 1,
+      uuid: "5FY8ghtsQaUJ2KzHJGetfE",
+      nickname: "Intern -1",
+    },
+  ];
+  const array_of_customerids = [
+    {
+      cust_id: 0,
+      uuid: "4vqyd62oARXqj9nRUNhtLQ",
+      api_key: "941532a0-6be1-443a-a9d5-d57bdf180a52",
+      nickname: "Customer account -1",
+      user_ids: array_of_userid_1,
+    },
+    {
+      cust_id: 1,
+      uuid: "6cBaidlJ84Ggc5JA7IYCgv",
+      api_key: "941532a0-6be1-443a-cdee-d57bdf180a52",
+      nickname: "Customer account -2",
+      user_ids: array_of_userid_2,
+    },
+  ];
+  const array_of_special_userids = [
+    {
+      cust_id: 0,
+      uuid: "4vqyd62oARXqj9nRUNhtLQ",
+      api_key: "941532a0-6be1-443a-a9d5-d57bdf180a52",
+      nickname: "Customer account -1",
+    },
+  ];
+  const array_of_special_customerid_null_userids = [
+    {
+      cust_id: 0,
+      uuid: "4vqyd62oARXqj9nRUNhtLQ",
+      api_key: "941532a0-6be1-443a-a9d5-d57bdf180a52",
+      nickname: "",
+    },
+  ];
+
+  test("When mounting SettingsForm from the build dist file, Then verify that it loads successfully", () => {
+    const propsData = null;
+    wrapper = mount(DistComponentToTest, {
+      propsData,
+      store,
+      localVue,
+    });
+
+    const target_span = wrapper.find(".span__settingsform-title");
+
+    expect(target_span.text()).toStrictEqual("Settings");
+  });
+
+  test("Given that the SettingsForm is loaded with Vuex store, When user action which generates an event 'cancel-id', Then the Vuex Store is not modified", async () => {
+    store.commit("settings/set_customer_account_ids", array_of_customerids);
+    store.commit("settings/set_customer_index", 1);
+
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+
+    wrapper.vm.onCancelCustomerId();
+    expect(store.getters["settings/customer_account_ids"]).toStrictEqual(
+      array_of_customerids
+    );
+    expect(store.getters["settings/customer_index"]).toStrictEqual(1); // this is the real data due to savechanges function Vuex stored data of customer_index
+  });
+  test("Given that the SettingsForm is loaded with Vuex store, When there is no data in Vuex, Then <empty> is set in the Input of Customer ID and User ID", () => {
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+    wrapper.vm.onCancelAddCustomerId(); // Complete Code Coverage
+    expect(
+      wrapper.find("#input-dropdown-widget-cust-").element.value
+    ).toStrictEqual("");
+    expect(
+      wrapper.find("#input-dropdown-widget-user-").element.value
+    ).toStrictEqual("");
+  });
+  test("Given that the SettingsForm is loaded with Vuex store, When there is no data in the user_ids, Customer Index/User Index is not set in the Vuex, Then <empty> is set in the Input of  Customer ID/User ID", () => {
+    store.commit("settings/set_customer_account_ids", array_of_special_userids);
```

Comment:
`userids` is not one word, should be `user_ids`

Reply:
Corrected